detekt.yml

```diff
@@ -278,7 +278,7 @@ naming:
     minimumFunctionNameLength: 3
   FunctionNaming:
     active: true
-    functionPattern: '^([a-z$][a-zA-Z$0-9]*)|(`.*`)$'
+    functionPattern: '^([a-z$A-Z][a-zA-Z$0-9]*)|(.*)$'
```

Comment:
I see for the last part at the files app, it is still using the ` quotes around  (.*)
Whats the difference?

https://github.com/nextcloud/android/blob/e3d57e7c42efa5dd4b52aeea71ee1dbbd903fe19/app/detekt.yml#L204